packages/console/src/components/Launch/LaunchForm/StructInput.tsx

```diff
@@ -69,7 +69,10 @@ export const StructInput: React.FC<InputProps> = props => {
       literalType?.metadata?.fields?.definitions?.structValue?.fields,
     );
 
-    if (keys[0]) {
+    if (keys.length > 1) {
+      // If there are multiple keys, we can't render a form because of not supporting nested structs
+      jsonFormRenderable = false;
```

Comment:
I think we want to render a free-form text area in this case; they can paste in JSON?